File: c program/factorial.c
#include<stdio.h>
int main()
{
    int num, i;
    int factorial =1;
    printf("ENTER THE NUMBER :");
    scanf("%d",&num);
    for(i=1;i<=num;i++)
    {
        factorial = factorial *i;
    }
    printf("THE FACTORIAL OF NUMBER %d is %d",num,factorial);

}

File: java program/factorial/src/factorial/Factorial.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package factorial;

import java.util.Scanner;

/**
 *
 * @author Ayush
 */
public class Factorial {

    /**
     * @param args the command line arguments
     */
    public static void main(String[] args) {
        int num;
        int factorial=1;
        Scanner s = new Scanner(System.in);
        System.out.println("Enter the number");
        num = s.nextInt();
        while (num != 0)
        {
          
           factorial = factorial * num;
           num = num -1;
           
        }
        System.out.println(factorial);
    } 
}

    

